lib/internal/bootstrap/node.js

```diff
@@ -599,21 +598,21 @@ function setupGlobalEncoding() {
 
 function setupQueueMicrotask() {
   Object.defineProperty(global, 'queueMicrotask', {
-    get: () => {
+    get() {
       process.emitWarning('queueMicrotask() is experimental.',
                           'ExperimentalWarning');
-      const { setupQueueMicrotask } =
+      const { queueMicrotask } =
         NativeModule.require('internal/queue_microtask');
-      const queueMicrotask = setupQueueMicrotask(triggerFatalException);
-      Object.defineProperty(global, 'queueMicrotask', {
+
+        Object.defineProperty(global, 'queueMicrotask', {
```

Comment:
Indentation is off and the linter is complaining.

```suggestion
      Object.defineProperty(global, 'queueMicrotask', {
```